src/components/Menu/MenuBar.vue

```diff
@@ -178,9 +159,7 @@ export default {
 		},
 	},
 	mounted() {
-		window.addEventListener('resize', this.getWindowWidth)
-		subscribe('files:sidebar:opened', this.redrawAfterTransition)
-		subscribe('files:sidebar:closed', this.redrawAfterTransition)
+		this.resize = useResizeObserver(this.$refs.menubar, this.onResize)
```

Comment:
Is there any reason to keep a handle on the resize observer?

Reply:
Right, I missed to clean up the resize observer. Documentation is not very specific if that needs to be done, but reading https://stackoverflow.com/questions/67581868/should-i-call-resizeobserver-unobserve-for-removed-elements it felt more safe to do so.